vault/resource_aws_auth_backend_client.go

```diff
@@ -78,6 +84,7 @@ func awsAuthBackendWrite(d *schema.ResourceData, meta interface{}) error {
 	ec2Endpoint := d.Get("ec2_endpoint").(string)
 	iamEndpoint := d.Get("iam_endpoint").(string)
 	stsEndpoint := d.Get("sts_endpoint").(string)
+	stsRegion := d.Get("sts_region").(string)
```

Comment:
Is there a situation where `sts_region` would be an empty string, and would that have an effect on the Vault side here https://github.com/hashicorp/vault/blob/314bc50de40afa8545c52849c55bb204b79e95fc/builtin/credential/aws/path_config_client.go#L239 ?